scripts/test.js

```diff
@@ -4,6 +4,27 @@ require("@app/config/env");
 const { execSync, spawnSync: rawSpawnSync } = require("child_process");
 const concurrently = require("concurrently");
 
+// Dear graphile-migrate, please treat the test DB as if it were the shadow DB
+process.env.SHADOW_DATABASE_URL = process.env.TEST_DATABASE_URL;
+
+// Signal to graphile-migrate scripts that we're in the tests
+process.env.IN_TESTS = "1";
+process.env.NODE_ENV = "test";
+
+const cmdArgs = process.argv;
```

Comment:
The first two args are `node` and the script name, so we should skip those

```suggestion
const cmdArgs = process.argv.slice(2);
```